app/test_main.py

```diff
@@ -1,3 +1,20 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize("cat_years, dog_years, expected", [
+    (0, 0, [0, 0]),
+    (13, 13, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (20, 20, [1, 1]),
+    (23, 23, [1, 1]),
+    (27, 26, [2, 2]),
+    (28, 28, [3, 2]),
+    (30, 30, [3, 3]),
+    (100, 100, [21, 17]),
+])
```

Comment:
Consider adding edge cases for negative numbers or extremely large numbers to ensure the function handles these scenarios correctly. This aligns with the checklist's recommendation to test edge situations.